iOS/SampleApp/SampleApp/Services/CurrentValue.swift

```diff
@@ -0,0 +1,25 @@
+// Copyright 2019 Flying Vehicle Monster team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
```

Comment:
Me neither, what's actual purpose of this struct?

Reply:
This structure saves some informations before lost them during someone uses FVM

Reply:
xCode can't get me to this struct's callers, could you point them out?

Reply:
For example: 
viewDidAppear() in ViewController 
prepareDamagedCarPartsToSend() in ManagingJSON

Actually in each SampleApp Swift's file you can find using of this struct